Exports/EventHandlers/PreInstallEventHandler/Item/PreInstallEventHandler.cs

```diff
@@ -1,17 +1,14 @@
-﻿using System;
-using System.Net;
-using System.Runtime.InteropServices;
-using kCura.EventHandler;
-using kCura.EventHandler.CustomAttributes;
-using kCura.Relativity.Client;
+﻿using kCura.EventHandler;
 using Relativity.API;
 using Relativity.Services.Objects;
+using System;
+using System.Net;
 
 namespace $rootnamespace$
 {
 	[kCura.EventHandler.CustomAttributes.Description("Pre Install EventHandler")]
 	[System.Runtime.InteropServices.Guid("$guid1$")]
-	public class $safeitemname$ : kCura.EventHandler.PreInstallEventHandler
+	public class $safeitemname$ : kCura.EventHandler.PreDeleteEventHandler
```

Comment:
Should be PreInstallEventHandler not PreDeleteEventHandler

Reply:
Fixed